src/sql/base/browser/ui/infoBox/infoBox.ts

```diff
@@ -96,6 +113,54 @@ export class InfoBox extends Disposable implements IThemable {
 		}
 	}
 
+	public get isClickable(): boolean {
+		return this._isClickable;
+	}
+
+	public set isClickable(v: boolean) {
+		if (this._isClickable === v) {
+			return;
+		}
+		this._isClickable = v;
+		if (this._isClickable) {
+			this._clickableIndicator.style.display = '';
+			this._clickableIndicator.tabIndex = 0;
+			this._infoBoxElement.style.cursor = 'pointer';
+			this._infoBoxElement.setAttribute('role', 'button');
+			this._textElement.style.maxWidth = 'calc(100% - 75px)';
+			this.registerClickListeners();
+		} else {
+			this._clickableIndicator.style.display = 'none';
+			this._clickableIndicator.tabIndex = -1;
+			this._infoBoxElement.style.cursor = 'default';
+			this._infoBoxElement.removeAttribute('role');
+			this._textElement.style.maxWidth = '';
+			this.unregisterClickListeners();
+		}
+	}
+
+	private registerClickListeners() {
+		this._onDidClick = this._clickListenersDisposableStore.add(new Emitter<undefined>());
```

Comment:
This is fine to initialize with the class. It just won't ever be fired unless `isClickable` is set to true. 

(with how you have it if someone changed isClickable to false and then back to true the events wouldn't fire since the infoBox component only hooks up to the initial emitter)